.github/release-please.yml

```diff
@@ -3,4 +3,4 @@ bumpMinorPreMajor: true
 handleGHRelease: true
 primaryBranch: main
 manifest: true
-extraFiles: ["WORKSPACE", ".cloudbuild/cloudbuild.yaml", ".cloudbuild/cloudbuild-test-a.yaml", ".cloudbuild/cloudbuild-test-b.yaml" ]
+extraFiles: ["WORKSPACE", ".cloudbuild/graalvm/cloudbuild.yaml", ".cloudbuild/graalvm/cloudbuild-test-a.yaml", ".cloudbuild/graalvm/cloudbuild-test-b.yaml" ]
```

Comment:
Can you please add this [file](https://github.com/googleapis/sdk-platform-java/blob/main/.cloudbuild/library_generation/cloudbuild-library-generation-release.yaml) as well? Same for `release-pkease-config.json`.